packages/caravan-health/src/waste.test.ts

```diff
@@ -129,12 +129,44 @@ describe("Waste metric scoring", () => {
   });
 
   describe("Dust Limits", () => {
-    it("calculates the lower and upper limit of the dust amount based on the fee rate and transaction weight", () => {
+    it("calculates the lower and upper limit of the dust amount for P2SH script type and 1.5 risk multiplier", () => {
       const uninitializedWasteMetric = new WasteMetrics();
       const { lowerLimit, upperLimit } =
-        uninitializedWasteMetric.calculateDustLimits(10, 300, 1.5);
-      expect(lowerLimit).toBe(4480);
-      expect(upperLimit).toBe(6720);
+        uninitializedWasteMetric.calculateDustLimits(10, "P2SH", 1.5);
```

Comment:
"uninitialized"? Does this even need to be on the class then?

Reply:
This one, Just to let users explore that the class doesn't necessarily need transactions and UTXO all the time. It still has some methods without it. So made this one test case such that it strikes their mind.